doc/herbstluftwm.txt

```diff
@@ -425,7 +425,7 @@ split 'ALIGN' ['FRACTION' ['FRAMEINDEX']]::
         * split h
         * split top 0.2 ''   (splits the root frame)
 
-focus ['-i'|'-e'] 'DIRECTION'::
+focus [*--level=*'LEVEL'] 'DIRECTION'::
```

Comment:
Oh wait, isn’t this a breaking change?

(Sorry if you’re already planning to handle it properly; didn’t want it to land in `master` unnoticed.)

Reply:
You're right! But the flags -i/-e are still handled, so I just added it to the documentation again.

Reply:
Ah, alrighty then. Thanks!